src/gui/MainWindow.cpp

```diff
@@ -464,6 +464,11 @@ void MainWindow::setMenuActionState(DatabaseWidget::Mode mode)
             m_ui->actionDatabaseMerge->setEnabled(m_ui->tabWidget->currentIndex() != -1);
 
             m_searchWidgetAction->setEnabled(true);
+
+            //if not already in search mode focus the search widget and select all its content
+            if (!inSearch) {
+                m_ui->toolBar->findChild<SearchWidget *>("SearchWidget")->searchFocus();
```

Comment:
Thank you for the reply and for the advice. I've a question: the unlockedDatabase() signal is emitted by DatabaseWidget.cpp so how can I use that signal in SearchWidget.cpp?